runtime/runtime/src/config.rs

```diff
@@ -107,7 +106,7 @@ pub fn total_send_fees(
                 transfer_send_fee(config, sender_is_receiver, is_receiver_implicit)
             }
             Stake(_) => config.fee(ActionCosts::stake).send_fee(sender_is_receiver),
-            AddKey(AddKeyAction { access_key, .. }) => match &access_key.permission {
+            AddKey(add_key_action) => match add_key_action.access_key.permission.clone() {
```

Comment:
Is cloning here really necessary? Seems like handling this by-reference should be possible.